certbot/display/util.py

```diff
@@ -164,6 +178,7 @@ def yesno(self, message, yes_label="Yes", no_label="No", **unused_kwargs):
         :rtype: bool
 
         """
+        assert OK == self.dialog.DIALOG_OK, "What kind of absurdity is this?"
```

Comment:
The reason this is here is the same as the reason we have these janky constants in the first place: they're not available from the `dialog` module, only from `dialog.Dialog` objects.  But maybe we should just make a Dialog object at import time...